pycbc/fft/fftw.py

```diff
@@ -30,8 +30,8 @@
 # We need to construct them directly with CDLL so
 # we can give the RTLD_GLOBAL mode, which we must do
 # in order to use the threaded libraries as well.
-double_lib = get_ctypes_library('fftw3',['fftw3'],mode=ctypes.RTLD_GLOBAL)
-float_lib = get_ctypes_library('fftw3f',['fftw3f'],mode=ctypes.RTLD_GLOBAL)
```

Comment:
Should the mode be set deeper, ie. in the `get_ctypes_library` function itself, so that it is used *all the time* when using ctypes??

Reply:
... Or make this flag the default??

Reply:
We should check, but I think this would make sense.  Certainly if we are putting logic to decide whether or not a given installation has (or should use) `RTLD_DEEPCOPY` versus `RTLD_GLOBAL`, then I think it makes sense to do that in only one place.  However I think I prefer the route of making it a default, so that we retain the ability to do something different on a library by library basis, in case we find we need to do something different elsewhere.